cmd/server/assets/login/login.html

```diff
@@ -112,6 +112,7 @@
           .then(function(userCredential) {
             loginSuccess();
           }).catch(function(error) {
+            console.error(error);
```

Comment:
do we want to always log this to the console?

Reply:
I can switch it to just unhandled to be more conservative. This was a suggestion from @sethvargo to make debugging easier